test/errors/useImportNotModule-detailed.good

```diff
@@ -0,0 +1,6 @@
+─── error in useImportNotModule.chpl:8 [UseImportNotModule] ───
+  Target 'someProc' of use is not a module
```

Comment:
This could probable be worded in a more user-friendly manner. "Attempting to {import,use} 'someProc', which is not a module". And then a `.message` call after the `.code`: "In Chapel, only modules can be used in 'import' or 'use' statements.".